popsycle/synthetic.py

```diff
@@ -1982,10 +1903,10 @@ def unique_blends(blend_table):
     Parameters
     ---------
     blend_table : blend array 
-        A table with all the events. There are 34 columns: 1 with the unique
+        A table with all the events. There is 1 column with the unique
         source ID, 1 with the unique lens ID lens, 1 with the lens-neighbor
-        separation, and 31 with info about the neighbor (same order as the 
-        other "all info" tables).
+        separation, and the remaining columns contain info about the neighbors
+        (same order as the other "all info" tables).
```

Comment:
(since it's by name, there is no order anymore, right?)

Reply:
Correct, that language is removed